subjects/java/checkpoints/monthly-period/README.md

```diff
@@ -0,0 +1,60 @@
+## Monthly Period
+
+### Instructions
+
+Create a class `MonthlyPeriod` that provides a method to calculate the period between two given dates in terms of months and years. The dates will be provided in the format `yyyy-MM-dd`.
```

Comment:
What is the expected format of the period?
I deduce from the example that it should be `x years and y months` so can the numbers of days be omitted?
Should the `years` be `year` (same for `months`) if it's only 1?

Reply:
The expected format is clearly showed in the examples. I'm agree with you if there is only 1 it should be `month`/`year`.

Reply:
What about the days? I see you're not supposed to return the days, but what if the difference is 1 year, 2 months and 30 days: is it supposed to be 1 year and 2 months or 1 year and 3 months (since it's closer to being 3 months than 2 months)? 
I feel adding an example or changing the first one by pushing the `endDate` of the first one to `2023-06-30` should already give enough information on how these cases should be handled

Reply:
the diff should be done using the standard library, so they will always get the right period.
here i asked just for months and years while the month is not completed they can't count it as a fulll month